### PR TITLE
Test framework

### PR DESCRIPTION
**Pull request by henryzbw**
Init test framework

### CODE REVIEW

**Comment by amazingandyyy** [comment]
`yarn.lock`?

**Comment by henryzbw** [comment]
removed

File: __test__/setup/setupEnzyme.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import Enzyme from 'enzyme';
```
Comment (dwalintukan) [inline __test__/setup/setupEnzyme.js]: can we rename the folder `__test__` to `just `test`? or does it need to have the underscores?

File: jest.config.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+  // Options that will be passed to the testEnvironment
+  // testEnvironmentOptions: {},
+
+  // Adds a location field to test results
+  // testLocationInResults: false,
+
+  // The glob patterns Jest uses to detect test files
+  // testMatch: [
+  //   "**/__tests__/**/*.js?(x)",
+  //   "**/?(*.)+(spec|test).js?(x)"
```
Comment (dwalintukan) [inline jest.config.js]: lets use this `testMatch` and name our test files like `AllEventsStore.test.js`

File: src/stores/UiStore.test.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+const navigatorMock = {
+  language: 'en-US',
+};
+global.navigator = navigatorMock;
+global.localStorage = localStorageMock;
+
+// start test
+describe('UiStore', () => {
+  it('change locale', () => {
+    const store = new UiStore();
```
Comment (dwalintukan) [inline src/stores/UiStore.test.js]: lets create a `store` and setup the instance beforeEach test.

```
describe('UiStore', () => {
  let store;

  beforeEach(() => {
    store = new UiStore();
  }

  // test cases
});
```

Comment (dwalintukan) [inline src/stores/UiStore.test.js]: lets create a `store` and setup the instance beforeEach test.

```
describe('UiStore', () => {
  let store;

  beforeEach(() => {
    store = new UiStore();
  }

  // test cases
});
```

**Pull request merged**